avocado_vt/plugins/vt_resolver.py

```diff
@@ -45,6 +45,8 @@ def _get_reference_resolution(self, reference):
             if self.config.get("nrunner.max_parallel_tasks", 1) != 1:
                 warnings.warn("The VT NextRunner can be run only with "
                               "nrunner-max-parallel-tasks set to 1")
+                return ReferenceResolution(reference,
```

Comment:
I think this specific bit does more harm than good, because:

1. Resolution happens earlier than execution, and possibly even without execution
2. It's looking at an execution time configuration
3. It makes it impossible to run tests in parallel -- IMO we should provide mechanisms, but not enforce policies here
4. `NOTFOUND` is a stretch of what's being signaled here

Reply:
Yes I understand your concerns, what about cancel the tests inside the vt-runner if the serial run is disabled? Because for now we can't run vt-tests in parallel.